src/AccordionContainer/AccordionContainer.spec.tsx

```diff
@@ -264,6 +333,83 @@ describe('Accordion', () => {
         );
     });
 
+    it('setting the expanded property to false in an accordion that allows zero expansions when there is only one item expanded closes that item', () => {
+        const mock = jest.fn(() => null);
+        const fooItem = {
+            ...DEFAULT_ITEM,
+            uuid: 'foo',
+            expanded: true,
+        };
+
+        const instance = mount(
+            <Provider allowZeroExpanded={true} items={[fooItem]}>
+                <Consumer>{mock}</Consumer>
+            </Provider>,
+        ).instance() as Provider;
+
+        instance.setExpanded(fooItem.uuid, false);
+
+        expect(mock).toHaveBeenCalledWith(
+            expect.objectContaining({
+                items: [expect.objectContaining({ expanded: false })],
+            }),
+        );
+    });
+
+    it("setting the expanded property to false in an accordion that doesn't allow zero expansions when there is only one item expanded doesn't close that item", () => {
```

Comment:
These tests raise some questions about whether or not we should support 'controlled' accordions (ie. the `expanded` prop). We basically have two conflicting mechanisms for state management, and one can be leveraged/abused to make the accordion not-accessible. Nothing to action here, just a mental note about reviewing the `expanded` prop functionality (will raise an issue/proposal).